tests/framework/filters/VerbFilterTest.php

```
@@ -0,0 +1,256 @@
+        $event = $this->mockActionEvent('update');
+
+        $filter = new VerbFilter([
+            'actions' => [
+                'update' => ['GET', 'PUT', 'POST'],
+            ],
+        ]);
+
+        try {
+            $filter->beforeAction($event);
+        } catch (MethodNotAllowedHttpException $e) {
+        }
```

PHPMD (2.15.0)

[error] Avoid using empty try-catch blocks in testAllowHeaderSetOnDisallowedVerb. (undefined)

(EmptyCatchBlock)